opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/services/EncryptionService.java

```diff
@@ -0,0 +1,77 @@
+package de.adorsys.opba.protocol.facade.services;
+
+import com.google.common.io.Resources;
+import com.google.crypto.tink.Aead;
+import com.google.crypto.tink.CleartextKeysetHandle;
+import com.google.crypto.tink.JsonKeysetReader;
+import com.google.crypto.tink.KeysetHandle;
+import com.google.crypto.tink.aead.AeadConfig;
+import com.google.crypto.tink.subtle.AesGcmJce;
+import lombok.SneakyThrows;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.springframework.stereotype.Service;
+
+import javax.crypto.SecretKey;
+import javax.crypto.SecretKeyFactory;
+import javax.crypto.spec.PBEKeySpec;
+import java.nio.file.Paths;
+import java.security.Provider;
+import java.security.Security;
+import java.util.Base64;
+
+@Service
+public class EncryptionService {
```

Comment:
Could you please make an interface out of it?
I think it's really the place where someone would like to use own implementation

Reply:
sure. done